tests/README.md

```diff
@@ -1,6 +1,21 @@
 Valkey Test Suite
 =================
 
+Overview
```

Comment:
We seem to never describe this in the README, do we need it? We currently put the content in runtest --help. IF we want to put something in the README, we may need to add more content (such as how to write tests, how to get familiar with tests, how to run tests, etc.)